about.html

```diff
@@ -159,7 +159,7 @@ <h2>Partner</h2>
   <footer class="lightFooter">
     <img src="./images/footerlogo.png" alt="TIG Web Conference Logo">
     <div>
```

Comment:
- [ ] Kindly make sure this logo is not broken :recycle: 

![Screenshot from 2023-02-09 10-19-42](https://user-images.githubusercontent.com/97834160/217757461-394bdace-bb8d-4186-b848-44a6eab0a9d3.png)